### PR TITLE
Issue (Use format strings throughout TARDIS)  Solved

### PR DESCRIPTION
**Pull request by yashagg2001**
Issue(Use format strings throughout TARDIS)  Solved:- 
I changed all string formatting from .format() to f-string formatting in all the .py files.
Please review it.
Thanks.

### CODE REVIEW

**Comment by andrewfullard** [comment]
There are 449 file changes in this PR. Can you fix it so only your actually changed files are shown? Thanks for your contribution!

**Comment by yashagg2001** [comment]
Sir,
I changed all the .py files in which string formatting needs to be changed.
Can you please help me how I will fix this?

Regards
Yash Aggarwal
Contact- 9992040848

**Comment by andrewfullard** [comment]
You need to move all your files out of the `tardis-master` folder. Your commit has made an entire copy of tardis _inside_ tardis itself.

File: ah_bootstrap.py [hunk 1]
```diff
@@ -361,9 +361,7 @@ def get_local_directory_dist(self):
         if not os.path.isdir(self.path):
             return
 
-        log.info('Attempting to import astropy_helpers from {0} {1!r}'.format(
-                 'submodule' if self.is_submodule else 'directory',
-                 self.path))
+        log.info(f'Attempting to import astropy_helpers from {'submodule' if self.is_submodule else 'directory'} {self.path!r}')
```
Comment (andrewfullard) [inline ah_bootstrap.py:364]: This is invalid syntax at the `'submodule' string part

**Comment by epassaro** [comment]
You should'n make changes to `ah_bootstrap.py` nor `setup.py`. Those are from `astropy` package template.

File: tardis/io/config_internal.py [hunk 2]
```diff
@@ -37,15 +35,11 @@ def get_data_dir():
...
-            "{line_stars} \n\n".format(
-                line_stars="*" * 80,
-                config_file=config_fpath,
-                default_data_dir=DEFAULT_DATA_DIR,
-            )
+            f"\n{"*" * 80}\n\nTARDIS will download different kinds of data (e.g. atomic) to its data directory {DEFAULT_DATA_DIR}\n\n"
+            f"TARDIS DATA DIRECTORY not specified in {config_fpath}:\n\n"
+            f"ASSUMING DEFAULT DATA DIRECTORY {DEFAULT_DATA_DIR}\n "
+            f"YOU CAN CHANGE THIS AT ANY TIME IN {config_fpath} \n\n"
+            f"{"*" * 80} \n\n"
```
Comment (andrewfullard) [inline tardis/io/config_internal.py:42]: This is invalid syntax. Please run `python setup.py develop` or use linting to check for syntax errors.

**Comment by andrewfullard** [comment]
Please fix all syntax errors in your changes using your local development environment. I recommend using a linter such as pylint.

**Pull request closed without merging**